managed-databases/postgresql-and-mysql/how-to/apply-maintenance.mdx

```diff
@@ -0,0 +1,50 @@
+---
+meta:
+  title: How to apply scheduled maintenance to a Database Instance
+  description: This page explains how to apply maintenance to a Database Instance
+content:
+  h1: How to apply scheduled maintenance to a Database Instance
+  paragraph: This page explains how to apply maintenance to a Database Instance
+tags: managed-database postgresql mysql database-instance maintenance
+dates:
+  validation: 2024-04-19
+  posted: 2024-04-19
+categories:
+  - managed-databases
+  - postgresql-and-mysql
+---
+
+From time to time your Scaleway Managed Databases have to undergo maintenance to make sure that your nodes are up-to-date and have all the tools necessary to maintain a healthy lifecycle. For example, your engine version might need to be upgraded to the latest available minor version, or certain patches might need to be implemented.
+
+These maintenance are set up, run, and scheduled by Scaleway, but you can select when to apply them to avoid interruptions during peak times.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [PostgreSQL or MySQL Database Instance](/managed-databases/postgresql-and-mysql/quickstart) scheduled for maintenance
+
+1. Click **PostgreSQL and MySQL** under **Managed Databases** on the side menu. A list of your Database Instances displays.
+2. Look for an orange info icon to identify which Database Instances are scheduled for maintenance.
+3. Click the Database Instance name or <Icon name="more" /> > **More info** to access the database information page.
+
+    If your Database Instance is scheduled for maintenance, you will see a banner on the top of the page informing you when the scheduled maintenance will take place. You can either wait for the maintenance to be applied automatically at the expected date and time, or you can apply it immediately.
+
+    <Message type="important">
+      Certain maintenance cannot be applied manually. In this case, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.
```

Comment:
```suggestion
      Certain maintenances cannot be applied manually. In this case, you will receive an email informing you of the details of the scheduled maintenance, such as the date, time, and on which Database Instance it will be applied. You can also check this information on the Database Instance information page of the Scaleway console.
```